### PR TITLE
Add @akaszynski to authors

### PR DESCRIPTION
**Pull request by banesullivan**

### CODE REVIEW

**Review by akaszynski — APPROVED** [review]
Thanks!

File: scooby/__init__.py [hunk 1]
```diff
@@ -40,7 +40,7 @@
 ]
 
 
-__author__ = 'Dieter Werthmüller & Bane Sullivan'
+__author__ = 'Dieter Werthmüller, Bane Sullivan, & Alex Kaszynski'
```
Comment (prisae) [inline scooby/__init__.py:43]: Just thinking out loud, maybe looking into the future. What about 

'Dieter Werthmüller, Bane Sullivan, Alex Kaszynski, and contributors'

or

'Dieter, Bane, Alex, and contributors'

Comment (banesullivan) [reply scooby/__init__.py:43]: I'm up for that

Comment (akaszynski) [reply scooby/__init__.py:43]:
```suggestion
__author__ = 'Dieter Werthmüller, Bane Sullivan, Alex Kaszynski, and contributors'
```

File: setup.py [hunk 2]
```diff
@@ -11,7 +11,7 @@
 setuptools.setup(
     name="scooby",
     version=__version__,
-    author="Dieter Werthmüller & Bane Sullivan",
+    author="Dieter Werthmüller, Bane Sullivan, & Alex Kaszynski",
```
Comment (banesullivan) [inline setup.py:14]:
```suggestion
    author="Dieter Werthmüller, Bane Sullivan, Alex Kaszynski, and contributors",
```

**Pull request merged**